omnibus/config/software/private-chef-ctl.rb

```diff
@@ -14,13 +14,16 @@
 # limitations under the License.
 #
 
-name "private-chef-ctl"
+name "chef-server-ctl"
 
-source path: "#{project.files_path}/private-chef-ctl-commands"
+source path: "#{project.files_path}/../../src/chef-server-ctl"
 
 license :project_license
 skip_transitive_dependency_licensing true
 
+dependency "postgresql96" # for libpq
+
+dependency "appbundler"
 dependency "highline-gem"
 dependency "sequel-gem"
```

Comment:
these gems should get moved into the Gemfile.lock to get accurately appbundle'd.

if there are c-libs in the omnibus sources those deps should get moved here.